src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/on_system_capability_updated_notification.cc

```diff
@@ -80,13 +80,13 @@ void OnSystemCapabilityUpdatedNotification::Run() {
       break;
     }
     case mobile_apis::SystemCapabilityType::VIDEO_STREAMING:
-      if (hmi_capabilities_.video_streaming_capability()) {
-        msg_params[strings::system_capability]
-                  [strings::video_streaming_capability] =
-                      *hmi_capabilities_.video_streaming_capability();
-      } else {
-        return;
+      if ((*message_)[strings::msg_params].keyExists(strings::app_id)) {
+        (*message_)[strings::params][strings::connection_key] =
+            (*message_)[strings::msg_params][strings::app_id];
+        (*message_)[strings::msg_params].erase(strings::app_id);
+        SendNotification();
```

Comment:
I think will be better to follow the common approach and send the notification after the switch block (https://github.com/LuxoftSDL/sdl_core/pull/65/files#diff-fcb68c8b7934208bb4c5509a9f361dffR250)

Reply:
@LitvinenkoIra I believe it isn't correct solution, because sending the notification after `switch` block is inside in the `for` loop, which sends the notification to an each registered application. But regarding to the video streaming capability we need to send only to specific application.

Reply:
@IGapchuk got it, thanks